applications/DEMApplication/custom_strategies/strategies/explicit_solver_continuum.cpp

```diff
@@ -644,6 +644,35 @@ namespace Kratos {
         KRATOS_CATCH("")
     }
 
+    void ContinuumExplicitSolverStrategy::BreakAlmostBrokenSpheres() {
+
+        KRATOS_TRY
+
+        const unsigned int maximum_allowed_number_of_intact_bonds = (unsigned int) GetModelPart().GetProcessInfo()[MAX_NUMBER_OF_INTACT_BONDS_TO_CONSIDER_A_SPHERE_BROKEN];
+
+        //#pragma omp parallel for
```

Comment:
Didn't this work in parallel? I think it should...

Reply:
Yes, my bad, corrected, thanks!